webroot/src/pages/LicensingDetail/LicensingDetail.ts

```diff
@@ -77,4 +257,40 @@ export default class LicensingDetail extends Vue {
 
         await this.$store.dispatch('license/getLicenseeRequest', { compact: this.compact, licenseeId });
     }
+
+    checkIfLicenseActive(license: License) {
+        let isLicenseActive = false;
+
+        if (license && license.statusState === LicenseStatus.ACTIVE) {
+            isLicenseActive = true;
+        }
+
+        return isLicenseActive;
+    }
```

Comment:
nit, but I like to prefix boolean functions with is, has, etc. I also think this is simple enough to make into a one-liner, e.g.

```suggestion
    isLicenseActive(license: License) {
        return license?.statusState === LicenseStatus.ACTIVE);
    }
```